### DISCUSSION TITLE
富文本的背景颜色为什么改不了？

### DISCUSSION THREAD

**Discussion by L1-corona** [hellof2e/quark-design#305] [Announcements]
组件仅仅提供了部分的修改css变量内容，另外适用深度deep也改不了，这是出于什么考虑？

**Comment by xxxsf** [comment]
刚看到，您指的是quarkd吗？
quarkd是通用型基础组件，如果要深度修改样式，那就不属于通用型基础组件了。
如果您指的是使用 quarkc 构建的业务组件，这取决于你自己，任何样式都可修改。
